rotkehlchen/tests/api/test_statistics.py

```diff
@@ -473,12 +473,11 @@ def test_query_wrap(
     """
     rotki = rotkehlchen_api_server.rest_api.rotkehlchen
     db = rotki.data.db
-
-    response = requests.post(  # Test that there is no failure for no data
+    assert_proper_sync_response_with_result(response=requests.post(  # Test that there is no failure for no data  # noqa: E501
```

Comment:
Hmm the pytest test is still using the word wrap. I see even the API name is using. We renamed it to event analysis. Can you also rename the API, the test and all related stuff to be consistent?

https://docs.rotki.com/usage-guides/statistic.html#event-analysis